### PR TITLE
Adding `xplique` from PyPI

### PR DESCRIPTION
**Pull request by sugatoray**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by sugatoray** [comment]
## Build Failure - Windows

Fails on Windows. Passes for Linux and OSX.

Since it is a noarch package, this should not be a problem.

**Comment by sugatoray** [comment]
PR is ready. Request for review/merging.

cc: @conda-forge/help-python @BastianZim

File: recipes/xplique/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  home: https://pypi.org/project/Xplique/
+  summary: Explanations toolbox for Tensorflow 2
+  license: MIT
+  license_file: LICENSE
+  description: |
+    🦊 <b>Xplique</b> (pronounced <i>\ɛks.plik\</i>) is a Python toolkit dedicated to explainability, currently based on Tensorflow.
+    The goal of this library is to gather the state of the art of Explainable AI to help you understand your complex neural network models.
+
+    PyPI: [https://pypi.org/project/Xplique/](https://pypi.org/project/Xplique/)
+
```
Comment (beckermr) [inline recipes/xplique/meta.yaml:51]: I'm skeptical unicode is a good idea here. Can we only use ascii please?

**Review by beckermr — CHANGES_REQUESTED** [review]
see above

File: recipes/xplique/meta.yaml (hunk 1 above)
Comment (sugatoray) [reply recipes/xplique/meta.yaml:51]: Ok. I will update it accordingly.

**Comment by sugatoray** [comment]
Made the changes.

File: recipes/xplique/meta.yaml [hunk 2]
```diff
@@ -44,7 +44,7 @@ about:
   license: MIT
   license_file: LICENSE
   description: |
-    🦊 <b>Xplique</b> (pronounced <i>\ɛks.plik\</i>) is a Python toolkit dedicated to explainability, currently based on Tensorflow.
+    🦊 <b>Xplique</b> is a Python toolkit dedicated to explainability, currently based on Tensorflow.
```
Comment (beckermr) [inline recipes/xplique/meta.yaml:47]:
```suggestion
    <b>Xplique</b> is a Python toolkit dedicated to explainability, currently based on Tensorflow.
```

**Pull request merged**